cirno/Geometry/LineLike.cs

```diff
@@ -15,6 +15,10 @@ public abstract class LineLike : IShape {
             return (P1 - P2).Cross(other.P1 - other.P2) < tolerance;
         }
 
+        public bool Perpendicular(Line other, float tolerance = 0.0001f) {
```

Comment:
`Line other` should be `LineLike other`

Reply:
Done! I mainly did this because ``Parallel`` also accepted ``Line`` as parameter. Shall I change that too?

Reply:
Sure, it was my mistake. Thanks.